cpu/kinetis/Makefile.include

```diff
@@ -26,8 +26,8 @@ RAM_START_ADDR := $(shell printf "0x%08x" $$(($(RAM_BASE_ADDR)-($(KINETIS_SRAM_L
 # Define ROM_LEN with a non arithmetic value as it must be
 # evaluated in `cortexm_common` without a shell context
 # The `K` is correctly handled by both the linker and `cortexm_common`.
-ROM_LEN = $(KINETIS_ROMSIZE)K
-RAM_LEN = $$(($(KINETIS_RAMSIZE) * 1024))
+ROM_LEN := $(KINETIS_ROMSIZE)K
```

Comment:
Add somwhere up:

```
ifneq (,$(filter riotboot_serial,$(USEMODULE)))
  RIOTBOOT_LEN ?= 0x2000
endif
```

Would need to be tested by the CI, but I think it needs to be 4k for most riotboot_serial compilations

Reply:
Actually, this would not work when flashing, see other suggestion above.

Reply:
@benpicco can you split this out? its an actual bug in RIOT currently.